<launch>
  <include file="$(find panda_moveit_config)/launch/panda_control_moveit_rviz.launch">
    <arg name="robot_ip" value="172.16.0.2" />
    <arg name="load_gripper" value="false" />

    <arg name="launch_rviz" value="false" />
  </include>

  <node name="realsense_node" pkg="bin_picking" type="realsense_node" output="screen" />
</launch>
